umake/frameworks/ide.py

```diff
@@ -26,11 +26,10 @@
 from contextlib import suppress
 from gettext import gettext as _
 import grp
-from io import StringIO
```

Comment:
I guess something wrong happened when you rebase on latest master. All the imports (addition or deletion) are a revert compared to when I moved VisualStudioCode the the IDE category.